### PR TITLE
19404 fixed newsletter issue

### PR DESCRIPTION
**Pull request by Dharmeshvaja91**
### Fixed Issues (if relevant)

1. magento/magento2#19404: When subscribing to newsletter, if email already exists, don't throw exception, but just warn without exception

### Manual testing scenarios (*)

1. Subscribe for the news latter with already register emial

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @Dharmeshvaja91. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php [hunk 1]
```diff
@@ -151,10 +151,7 @@ public function execute()
                 $status = (int) $this->_subscriberFactory->create()->subscribe($email);
                 $this->messageManager->addSuccessMessage($this->getSuccessMessage($status));
             } catch (LocalizedException $e) {
-                $this->messageManager->addExceptionMessage(
-                    $e,
-                    __('There was a problem with the subscription: %1', $e->getMessage())
-                );
+                $this->messageManager->addErrorMessage(__('This email address is already subscribed.'));
```
Comment (sivaschenko) [inline app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php]: I think the throw exception on line 146 should be replaced with this messageManager call. Here several different exceptions are possible (i.e. validateEmailFormat also throws LocalizedException)

**Review by sivaschenko — COMMENTED** [review]
Thanks for your contribution. Please see the review notes. Also, please note that this issue is already fixed in https://github.com/magento/magento2/pull/19406

**Comment by sivaschenko** [comment]
@Dharmeshvaja91 thanks for the contribution. Unfortunately, I have to ask you to close this pull request as it is a duplicate. The issue is already fixed in https://github.com/magento/magento2/pull/19406

File: app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php (hunk 1 above)
Comment (Dharmeshvaja91) [reply app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php]: @sivaschenko yes you are right, I will change this too.

**Comment by Dharmeshvaja91** [comment]
> @Dharmeshvaja91 thanks for the contribution. Unfortunately, I have to ask you to close this pull request as it is a duplicate. The issue is already fixed in #19406

@sivaschenko thank you for your comments on this i have just checked solution as per this pull request #19406  but still i m getting the same issue.

As per you instruction i have updated my code as well 
please update me that should i continue work on this ?.

**Comment by sivaschenko** [comment]
Your solution (after the fix) looks better, however, the https://github.com/magento/magento2/pull/19406 was created earlier. I propose you to help with https://github.com/magento/magento2/pull/19406 finalization (i.e. comment that you still able to reproduce the issue and link this pull request as an example of fix). We will not be able to merge both pull requests

**Comment by Dharmeshvaja91** [comment]
@sivaschenko He have closed his pull request, so please accept my pull request.

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-3574](https://jira.corp.magento.com/browse/ENGCOM-3574) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@Dharmeshvaja91 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by magento-engcom-team** [comment]
Hi @Dharmeshvaja91. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**